.gitignore

```
@@ -33,4 +33,4 @@ yarn-error.log*
 # vercel
 .vercel
 
-.env
+.env
```

not sure what the purpose of this change was 😄 
```suggestion
.env

```